lnwallet/channel.go

```diff
@@ -7675,19 +7687,38 @@ type LocalForceCloseSummary struct {
 	// then this will be nil.
 	CommitResolution *CommitOutputResolution
 
+	// AnchorResolution contains the data required to sweep the anchor
+	// output. If the channel type doesn't include anchors, the value of
+	// this field will be nil.
+	AnchorResolution *AnchorResolution
+
 	// HtlcResolutions contains all the data required to sweep any outgoing
 	// HTLC's and incoming HTLc's we know the preimage to. For each of these
 	// HTLC's, we'll need to go to the second level to sweep them fully.
 	HtlcResolutions *HtlcResolutions
+}
 
-	// ChanSnapshot is a snapshot of the final state of the channel at the
-	// time the summary was created.
-	ChanSnapshot channeldb.ChannelSnapshot
+// ForceCloseOpt is a functional option argument for the ForceClose method.
+type ForceCloseOpt func(*forceCloseConfig)
 
-	// AnchorResolution contains the data required to sweep the anchor
-	// output. If the channel type doesn't include anchors, the value of
-	// this field will be nil.
-	AnchorResolution *AnchorResolution
+// forceCloseConfig holds the configuration options for force closing a channel.
+type forceCloseConfig struct {
+	// includeResolutions if true will include the HTLC resolutions in the
+	// returned summary.
+	excludeResolutions bool
+}
+
+// defaultForceCloseConfig returns the default force close configuration.
+func defaultForceCloseConfig() *forceCloseConfig {
+	return &forceCloseConfig{}
+}
+
+// WithExcludeResolutions creates an option to exclude the force close
+// resolutions from the returned summary.
+func WithExcludeResolutions() ForceCloseOpt {
```

Comment:
Just suggesting some naming, feel free to ignore: `WithSkipResolution` (and `skipResolution bool` above)? Since it's more the action that's important to us rather than what's returned, right?